src/SDKs/Azure.Core/data-plane/Azure.Core/OptionsStore.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Azure.Core
+{
+    struct OptionsStore
+    {
+        // TODO (pri 3): optimize the dictionaries
```

Comment:
Do we have issues filed for these TODO's?

Reply:
Sort of -- we have a high level tracking work item to address the TODOs. After the PR is merged (or maybe even before, depending on the number of issues) I will go ahead and scrub the code base and update the high level work item with concrete details.

Reply:
I think I will keep the TODOs in the code even if they are already in the issue as well -- I feel like that will make it easier to pick off one at a time and see the context of the TODO.